modules/rhel-compute-requirements.adoc

```diff
@@ -52,7 +52,7 @@ link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index
 and the `disk.enableUUID=true` attribute must be set.
 
 * Each system should be able to access the cluster's API
-endpoints using a DNS resolvable hostname. Any network security access control in place should allow the system access to the
+endpoints by using  DNS resolvable hostnames. Any network security access control in place should allow the system access to the
```

Comment:
`endpoints by using  DNS resolvable hostnames. Any network security access control in place should allow the system access to the`

->

`endpoints by using DNS resolvable hostnames. Any network security access control that is in place must allow the system access to the`

Reply:
Sounds good